src/NServiceBus.Persistence.ServiceFabric.Tests/Tests.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
What's the reason to make `<OutputType>` an exe rather than dll?

Reply:
See above